pages/common/plesk.md

```diff
@@ -0,0 +1,28 @@
+# plesk
+
+> Plesk hosting control panel CLI interface.
+> More information: <https://docs.plesk.com/>.
+
+- Generate auto login link for admin user:
+
+`plesk login`
+
+- Show product version information:
+
+`plesk version`
+
+- Show list of hosted domains:
+
+`plesk bin domain --list`
+
+- Start watching for changes in panel.log file:
```

Comment:
```suggestion
- Start watching for changes in the panel.log file:
```

Reply:
I'd also add backticks around `panel.log` because it's a filename.